docs/x509/reference.rst

```diff
@@ -415,6 +463,29 @@ X.509 Certificate Object
         certificate validation is a complex problem that involves much more
         than just signature checks.
 
+        An example on how to check if a certificate signature is correct with a method
+        on the public key (this does not mean that the cert should be truster):
+
+        .. doctest::
+
+           >>> from cryptography.hazmat.primitives.serialization import load_pem_public_key
+           >>> from cryptography.exceptions import InvalidSignature
+           >>> from cryptography.hazmat.primitives.asymmetric import padding
+           >>> issuer_public_key = load_pem_public_key(pem_issuer_public_key, default_backend())
+           >>> cert_to_check = x509.load_pem_x509_certificate(pem_data_to_check, default_backend())
+           >>> try:
```

Comment:
For the doctest we can just call the verify call directly, no need to pu tit in a try block.

Reply:
hum ok, will change it
sorry for the failed CI, I made a copy/paste error...